ai-assistant-product/README_DE.md

```diff
@@ -1 +1,667 @@
+    - Beispiel:
+        - Benutzereingabe: „find my sick leave task“
+        - Sie haben ein Werkzeug, um Aufgaben nach Name, Beschreibung, Priorität usw. zu finden, aber der Benutzer hat nicht angegeben, welches Feld er verwenden möchte. Daher sollten Sie die Nachricht umformulieren, bevor Sie sie mit dem Ivy-Werkzeug verwenden.
+        - → „find task with name ‘sick leave’“
+- **customInstruction**: Bietet spezifische Richtlinien dafür, wie die KI die Nachricht umformulieren soll. Dies hilft der KI, besondere Fälle wie abstrakte Begriffe oder Datumsangaben zu behandeln und sie in nützlichere Datenformate zu bringen.
+
+- **onRephrase**: Definiert den nächsten Schritt, der ausgeführt wird, wenn die KI feststellt, dass die Nachricht umformuliert werden muss.
+
+- **onSuccess**: Gibt den Schritt an, der ausgeführt werden soll, wenn die KI die Nachricht nicht umformulieren muss.
+
+- **examples**: Eine Liste vordefinierter Beispiele, die der KI helfen, zu verstehen, wie Benutzernachrichten umformuliert werden sollen. Jedes Beispiel besteht aus:
+    - **before**: Die ursprüngliche, unstrukturierte Nachricht des Benutzers.
+    - **after**: Die umformulierte Nachricht, die die KI erzeugen würde, um mehr Klarheit und Handlungsfähigkeit zu gewährleisten.
+
+```json
+{
+    "stepNo": 0,
+    "type": "RE_PHRASE",
+    "useConversationMemory": true,
+    "toolId": "find-processes",
+    "onRephrase": 3,
+    "onSuccess": 1,
+    "customInstruction": "If in the message has an abstract date such as today, tomorrow,..., please format it. Example: today = 31, July 2024",
+    "examples": [
+        {
+            "before": "find leve request process",
+            "after": "find process that help creating leave request"
+        },
+        {
+            "before": "find process leave request",
+            "after": "find processes that have name 'leave request'"
+        },
+        {
+            "before": "find process 123",
+            "after": "find processes that the id is '123' or the name is '123'"
+        }
+    ]
+}
+```
+
+##### Trigger-Flow-Schritt
+
+Der **Trigger-Flow-Schritt** startet einen neuen Flow innerhalb des KI-Prozesses, entweder durch Übermittlung einer spezifischen Auslöse-Nachricht oder durch Nutzung des Ergebnisses eines vorherigen Schritts. Dies ermöglicht nahtlose Übergänge zwischen verschiedenen Workflows und die Weitergabe relevanter Daten zwischen ihnen.
+
+- **flowId**: ID des Flows, den Sie auslösen möchten.
+
+**Auslösen mit Auslöse-Nachricht**
+
+- **triggerMessage**: Die benutzerdefinierte Nachricht, die als Eingabe für den neuen Flow dient und die KI anleitet, was als Nächstes zu tun ist.
+
+```json
+{
+    "stepNo": 15,
+    "type": "TRIGGER_FLOW",
+    "flowId": "choose-member-flow",
+    "triggerMessage": "I want to choose members for my project described above"
+}
+```
+
+**Auslösen mit dem Ergebnis eines anderen Schritts**
```

Comment:
Do not understand this sentence.

Reply:
scenario: when user running AI Flow 1, the AI could perform some steps before decided that it should trigger another AI Flow 2 which is better to handle the request, and we would like to send the result which AI Flow 1 analyzed before hand which I put in a step.

Alternatives:
Auslösen mit dem Ergebnis eines anderen Schrittes
Trigger mit dem Ergebnis eines anderen Schrittes

Please help me choose 1 of those alternatives or suggest something better, thank you :)